uui-editor/src/SlateEditor.module.scss

```diff
@@ -1,5 +1,35 @@
 @use '~@epam/assets/scss/promo/colors' as *;
 @use '~@epam/assets/scss/promo/fonts' as *;
+@use '~@epam/assets/scss/promo/typography' as *;
+
+
+:local(.typography-16) {
+    @include typography-16();
+}
+
+:local(.typography-14) {
+    @include typography-14();
+}
+
+:local(.typography-header) {
+    @include typography-header();
+}
+
+:local(.typography-block) {
+    @include typography-block();
+}
+
+:local(.typography-inline) {
+    @include typography-inline();
+}
+
+:local(.typography-promo) {
+    font-family: $font-sans;
+    color: $gray80;
+    composes: typography-header;
+    composes: typography-block;
+    composes: typography-inline;
+}
```

Comment:
let add global mixin and use it all places:
 .typography-promo() {
    font-family: $font-sans;
    color: $gray80;
    composes: typography-header;
    composes: typography-block;
    composes: typography-inline;
}

Reply:
Done